system-tests/test/web_security_spec.js

```diff
@@ -83,6 +83,7 @@ describe('e2e web security', () => {
       },
       snapshot: true,
       browser: ['chrome', 'electron'],
+      expectedExitCode: 0,
```

Comment:
FYI, this is the default, so it doesn't need to be explicitly specified.

Reply:
sweet, I'll remove it!